airflow/timetables/interval.py

```diff
@@ -33,22 +34,14 @@
 Delta = Union[datetime.timedelta, relativedelta]
 
 
-class _DataIntervalTimetable(Timetable):
+class _CronAndDeltaTimetable(Timetable, ABC):
```

Comment:
We don’t use ABC here since it’s not necessary and incurs additional runtime cost.

Reply:
Thanks. Fixed by https://github.com/apache/airflow/pull/23662/commits/cb61d2e9c2fd7f9b30e7df2627cf06efbe53bfd8